### PR TITLE
Fix #798: Removed signal related files and depentant methods

### LINKED ISSUE #798: Remove Signals for building using the Heroku Build API
as per #749, and with #656 landed, the home page now pulls in content via XHR, not during the npm build step.

It should be safe to pull out this functionality now.

### PR DESCRIPTION
**Pull request by jayyyin**
Signals removed,
tester changed,
build and decorator files removed
fixes https://github.com/mozilla/network/issues/798

### CODE REVIEW

**Comment by Pomax** [comment]
Hi @jayyyin, can you explain what issue this PR addresses?

**Comment by jayyyin** [comment]
@Pomax yep it's this one https://github.com/mozilla/network/issues/798

**Comment by humphd** [comment]
@Pomax, @cadecairos could one of you take a look at this PR?  @jayyyin is my student, and this is for the course.  It would be great if someone could do a review.

**Comment by cadecairos** [comment]
@jayyin apologies for the silence. We added Snyk to this project to help monitor our dependencies, but due to the nature of the integration, PRs from contributors always fail on Travis and Appveyor. 

Since this isn't ideal, or helpful for contributors like yourself, I'm going to remove the automatic running of Snyk from the test scripts, which will unblock you once you rebase.

**Comment by jayyyin** [comment]
@cadecairos ok, so I just need to update my version is that correct?

**Comment by cadecairos** [comment]
@jayyyin yup, now that #852 has landed, a rebase should unblock merging being blocked

File: network-api/app/networkapi/highlights/apps.py [hunk 1]
```diff
@@ -6,5 +6,4 @@ class HighlightsConfig(AppConfig):
     verbose_name = 'highlights'
```
Comment (cadecairos) [inline network-api/app/networkapi/highlights/apps.py]: we should be able to remove this file entirely, if we remove the contents (but not the file) of `network-api/app/networkapi/highlights/__init__.py`

File: network-api/app/networkapi/news/apps.py [hunk 2]
```diff
@@ -6,5 +6,4 @@ class NewsConfig(AppConfig):
     verbose_name = 'news'
 
     def ready(self):
-        from networkapi.news.signals import setup_signals
-        setup_signals()
+        pass
```
Comment (cadecairos) [inline network-api/app/networkapi/news/apps.py]: we should be able to remove this file entirely, if we remove the contents (but not the file) of network-api/app/networkapi/news/__init__.py

File: network-api/app/networkapi/people/apps.py [hunk 3]
```diff
@@ -6,5 +6,4 @@ class PeopleConfig(AppConfig):
     verbose_name = 'people'
 
     def ready(self):
-        from networkapi.people.signals import setup_signals
-        setup_signals()
+        pass
```
Comment (cadecairos) [inline network-api/app/networkapi/people/apps.py]: we should be able to remove this file entirely, if we remove the contents (but not the file) of network-api/app/networkapi/people/__init__.py

File: network-api/app/networkapi/utility/tests.py [hunk 4]
```diff
@@ -1,56 +1,5 @@
...
-        test_fn(self)
-        sleep(2)
-        self.assertEqual(self.times_called, 1)
-
-        # call the function again, but the timeout should prevent execution
-        test_fn(self)
-        sleep(2)
-        self.assertEqual(self.times_called, 1)
+class TestUtilty(TestCase):
+    pass
```
Comment (cadecairos) [inline network-api/app/networkapi/utility/tests.py]: lets drop this file entirely

**Comment by humphd** [comment]
@jayyyin we haven't covered rebasing yet, so here is what you want to do (NOTE: I assume you have a `git remote` setup for Mozilla's repo named `upstream`, so change that if yours is named something else):

```
git checkout master
git pull upstream master
git checkout RemovingSignals
git rebase master
```

If you get conflicts at this stage, you can do `git rebase --abort` and it will roll things back, until I can talk to you and show you how to proceed.

If it works, you can push your code here (note the `-f` to force it to overwrite the history of this branch):

```
git push origin RemovingSignals -f
```

Let me know if you have questions.

**Comment by jayyyin** [comment]
@cadecairos changes have been done and committed, conflicts have been fixed via rebasing and everything looks good to go =D

**Comment by alanmoo** [comment]
@cadecairos is this good to go?

**Comment by jayyyin** [comment]
@cadecairos did I miss anything?

**Comment by jayyyin** [comment]
@cadecairos just realized I forgot to commit before pushing, last time I committed, silly me..., everything should be good now.

**Comment by alanmoo** [comment]
Looks like this is still failing. Is this based from the latest build of master?

**Comment by jayyyin** [comment]
@alanmoo that's probably the issue, let me fix that

**Comment by jayyyin** [comment]
@alanmoo pretty sure I rebased the master correctly and it appears at least one of the tests still fails, not sure what could be wrong now...

**Comment by alanmoo** [comment]
It’s a problem on master. This should be ready for a proper review when another PR lands and we rebase.

**Comment by jayyyin** [comment]
@alanmoo ah I see, so I just need to wait now?

**Comment by alanmoo** [comment]
Checks have passed. @cadecairos is this good now?

**Comment by jayyyin** [comment]
so I've been chatting with @humphd, and have heard that a lot of people have been away, I'd like to follow up with this.

**Comment by alanmoo** [comment]
I believe we've changed the folder structure a bit since this PR was opened, so it's going to take a bit of git manipulation to get this in merge-able shape again. You may want to open another branch off the current master and manually make the changes again.

**Comment by jayyyin** [comment]
@alanmoo how would I make a new branch, rebase the fork I have? or some re-fork it?

**Comment by jayyyin** [comment]
@alanmoo @cadecairos created new pull request here https://github.com/mozilla/foundation.mozilla.org/pull/920

**Pull request closed without merging**